x-pack/test/alerting_api_integration/spaces_only/tests/index.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SpacesService } from '../../../common/services';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { SpaceScenarios } from '../scenarios';
+
+// eslint-disable-next-line import/no-default-export
+export default function alertingApiIntegrationTests({
+  loadTestFile,
+  getService,
+}: FtrProviderContext) {
+  const spacesService: SpacesService = getService('spaces');
+  const esArchiver = getService('esArchiver');
+
+  describe('alerting api integration', function() {
```

Comment:
Should indicate spaces only. Change security_and_spaces test suite title as well.